pkg/components/kubescheduler/component.go

```diff
@@ -26,6 +26,14 @@ var KubeSchedulerComponent = Component{
 				SIG:      "sig-scheduling",
 				Priority: -1,
 			},
+			{Suite: "Scheduler predicates and priority test suites"},
+			{Suite: "Scheduler related scenarios"},
+			{Suite: "Testing Scheduler Operator related scenarios"},
+			{Suite: "nodeAffinity"},
+			{Suite: "podAffinity"},
+			{Suite: "resouces related scenarios"},
+			{Suite: "taint toleration related scenarios"},
+			{Suite: "Scheduler alert related features"},
```

Comment:
Please help add `IncludeAll: []string{"Workloads test kubescheduler operator works well"}`
`IncludeAll:[]string{"Workloads test predicates and priority work well}`
`IncludeAll:[]string{"Workloads taints and toleration works well"}`
`IncludeAll:[]string{"Workloads podTopologySpreadConstraints works well}`